micro-rdk/src/common/conn/viam.rs

```diff
@@ -502,6 +502,13 @@ where
         // if wifi manager is configured loop forever until wifi is connected via
         // a the provisioned network or one from previously stored agent config
         if let Some(wifi) = self.wifi_manager.as_ref().as_ref() {
+            if !self.storage.has_network_settings() {
+                log::info!("initializing additional networks storage...");
+                if let Err(e) = self.storage.store_network_settings(Default::default()) {
+                    log::error!("failed to initialize new network settings storage: {}", e);
```

Comment:
Make this error text align with whatever you pick above.

Reply:
yup, just noticed and made that change